packages/guui/components/Dropdown/index.js

```diff
@@ -229,20 +240,22 @@ export default class Dropdown extends Component<
 
                 <ul
                     id={dropdownID}
-                    className={this.state.isExpanded ? 'expanded' : ''}
+                    className={this.state.isExpanded ? ulExpanded : ul}
                 >
-                    {links.map(link => (
-                        <li key={link.title}>
+                    {links.map((l, index) => (
+                        <li key={l.title}>
                             <a
-                                href={link.url}
-                                className={link.isActive ? 'active' : ''}
+                                href={l.url}
+                                className={`${l.isActive ? linkActive : link} ${
```

Comment:
As per my previous comment, this looks like a use case for using `cx` to conditionally render class names.

Reply:
Definitely agree that double-ternary as here is not great.